core/Utils.vala

```diff
@@ -412,8 +445,8 @@ namespace Maya.Util {
     public bool is_multiday_event (iCal.Component comp) {
         var start = ical_to_date_time (comp.get_dtstart ());
         var end = ical_to_date_time (comp.get_dtend ());
-        start= start.to_timezone (new TimeZone.utc ());
-        end = end.to_timezone (new TimeZone.utc ());
+//        start= start.to_timezone (new TimeZone.utc ());
```

Comment:
Please remove code instead of commenting it out. Let the diff show what used to be there.